o1vm/src/mips/constraints.rs

```diff
@@ -374,75 +373,95 @@ impl<Fp: Field> InterpreterEnv for Env<Fp> {
         _len: &Self::Variable,
```

Comment:
Would be nice in the near future to add documentation on what `request_preimage_write` is supposed to do, on the constraint side.